src/test/java/de/tum/in/www1/artemis/tutorialgroups/TutorialGroupScheduleIntegrationTest.java

```diff
@@ -96,6 +96,35 @@ void createNewTutorialGroupWithSchedule_sessionFallsOnTutorialGroupFreeDay_shoul
         this.assertScheduledSessionIsCancelledOnDate(holidaySession, secondAugustMonday, persistedTutorialGroupId, persistedSchedule);
     }
 
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    void createNewTutorialGroupWithSchedule_wrongDateFormatInSchedule_shouldReturnBadRequest() throws Exception {
```

Comment:
You could think about converting this to a @ParameterizedTest

Reply:
Not really needed imo as text exercise is just a standing for all other entities. The concrete entity that is indirectly connected to the configuration is not important here.